go.mod

```
@@ -4,11 +4,12 @@ go 1.16
 
 require (
 	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
-	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.21.0
+	github.com/gardener/etcd-druid v0.5.0
+	github.com/gardener/gardener v1.23.0
```

```suggestion
	github.com/gardener/gardener v1.23.1
```

in order to get https://github.com/gardener/gardener/pull/4089?